daemon/cmd/integration_runner/main.go

```diff
@@ -245,6 +245,16 @@ func catRequest(w http.ResponseWriter, r *http.Request) {
 	w.Write(body)
 }
 
+func delayRequest(w http.ResponseWriter, r *http.Request) {
+	duration := r.URL.Query().Get("duration")
+	io.WriteString(w, "waiting...")
```

Comment:
Is this output necessary?

Reply:
It is not required for the guzzle tests.

Reply:
```suggestion
	io.WriteString(w, "waiting...")
```

Reply:
That suggestion is reasonable?

Reply:
After discussion it was clarified for me this output appears in the body of the response from the `/delay` endpoint and not to the console and so I am not asking for any changes.